client/src/containers/signup/Signup.css

```diff
@@ -54,3 +54,11 @@ limitations under the License.
 .next-name-signup-hint {
   color: var(--text-light) !important;
 }
+
+#next-signup-username-error {
+  color:#ffff;
```

Comment:
Nit: Convention is either 3 or 6 characters for hex code truncation. Alternatively, you can use `white`, which is primarily what the rest of the app does. Also, watch space between hash and colon.

Reply:
addressed